packages/oas-normalize/tsconfig.json

```diff
@@ -1,17 +1,12 @@
 {
+  "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "allowJs": true,
     "baseUrl": "./src",
-    "declaration": true,
-    "esModuleInterop": true,
-    "module": "ESNext",
-    "moduleResolution": "bundler",
+    "lib": ["DOM"],
```

Comment:
could we stick with the base `lib` value? why does this library need DOM APIs?

```suggestion
```

Reply:
`oas-normalize` uses `fetch` which only exists in `DOM`.